src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,58 @@
+.clock {
+  @include rectangle ($clock-size, $clock-size);
+  background-color: gold;
+  border-radius: 40%;
+
+  position: relative;
+
+  &__round {
+    top: 50%;
+
+    @include rectangle ($round-clock, $round-clock);
+    @extend %watch-hands;
+    background-color: orangered;
+    z-index: 1;
+    transform: translate(-50%, -50%);
```

Comment:
It better works with 'transform: translate(-25%, -50%);'

Reply:
fixed